spec/system/candidate_interface/references/candidate_reviews_references_spec.rb

```diff
@@ -105,15 +105,21 @@ def and_i_can_review_my_references_before_submission
       expect(all('.app-summary-card')[0].text).to have_content @requested_reference.email_address
       expect(all('.app-summary-card')[0]).not_to have_link 'Change'
       expect(all('.app-summary-card')[0]).not_to have_link 'Delete'
-      expect(all('.app-summary-card')[1].text).to have_content @refused_reference.email_address
+    end
+
+    within '#references_bounced' do
+      expect(all('.app-summary-card')[0].text).to have_content @bounced_reference.email_address
+      expect(all('.app-summary-card')[0]).not_to have_link 'Change'
+      expect(all('.app-summary-card')[0]).to have_link 'Delete request'
+    end
+
+    within '#references_failed' do
+      expect(all('.app-summary-card')[0].text).to have_content @refused_reference.email_address
+      expect(all('.app-summary-card')[0]).not_to have_link 'Change'
+      expect(all('.app-summary-card')[0]).to have_link 'Delete request'
+      expect(all('.app-summary-card')[1].text).to have_content @cancelled_reference.email_address
```

Comment:
Could you use the new funky summary card matcher here?

Reply:
Not yet, as those links are in the header of the summary card, not the summary list itself, which the matcher doesn't support... 🤦 

One for a follow-up PR though! :D